README.md

```diff
@@ -230,6 +230,8 @@ Ruby:
 Rust:
 
 - rustc
+- cargo (call with `Neomake! cargo`)
+- [clippy](https://github.com/Manishearth/rust-clippy) (call with `Neomake! clippy`, needs nightly rust, supports [rustup](https://github.com/rust-lang-nursery/rustup.rs))
```

Comment:
Probably won't, at least not soon.
Here's the developer saying it: https://www.reddit.com/r/rust/comments/4le45w/clippy_on_stable/d3mnrxe
Basically, clippy needs access to compiler internals that can only be accessed with a nightly compiler, since they are inherently not stable.